templates/eap73-image-stream.json

```diff
@@ -0,0 +1,177 @@
+{
+    "kind": "List",
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "jboss73-image-streams",
```

Comment:
File is named `eap73-image-stream` while resource name is `jboss73-image-streams`. Shouldn't they match?

Reply:
It's not appearing in the file name or resource name but this image stream is for JDK8. Should this be made explicit?
Do we need an image stream for JDK11 too?

Reply:
The convention in the past has been to pluralize the list name in the file:

https://github.com/jboss-container-images/jboss-eap-7-openshift-image/blob/7.2.x/templates/eap72-image-stream.json#L5

The JDK 8 / RHEL 7 templates and imagestreams don't container -openjdk8 as part of the name, as that was the convention for 7.2, and what the docs specify.

They currently both us -imagestreams in the names.

There is already an 7.3 OpenJDK 11 version (templates/eap73-openjdk11-image-stream.json).

Reply:
There is already an imagestream for JDK 11 (templates/eap73-openjdk11-image-stream.json)

The naming of the imagestream list within the file matches what we had done previously and is usually pluralized. (https://github.com/jboss-container-images/jboss-eap-7-openshift-image/blob/7.2.x/templates/eap72-image-stream.json#L5)